ios-drag-drop.js

```diff
@@ -326,17 +326,9 @@ function _exposeIosHtml5DragDropShim(config) {
   function touchstart(evt) {
     var el = evt.target;
     do {
-      if (el.draggable === true) {
-        // If draggable isn't explicitly set for anchors, then simulate a click event.
-        // Otherwise plain old vanilla links will stop working.
-        // https://developer.mozilla.org/en-US/docs/Web/Guide/Events/Touch_events#Handling_clicks
-        if (!el.hasAttribute("draggable") && el.tagName.toLowerCase() == "a" && config.simulateAnchorClick) {
-          var clickEvt = document.createEvent("MouseEvents");
-          clickEvt.initMouseEvent("click", true, true, el.ownerDocument.defaultView, 1,
-            evt.screenX, evt.screenY, evt.clientX, evt.clientY,
-            evt.ctrlKey, evt.altKey, evt.shiftKey, evt.metaKey, 0, null);
-          el.dispatchEvent(clickEvt);
-          log("Simulating click to anchor");
+      if (elementIsDraggable(el)) {
+        if (elementIsAnchor(el)) {
```

Comment:
This `if (elementIsAnchor(el))` check seems redundant. We could just call `handleTouchStartOnAnchor(el);` because that function contains the `elementIsAnchor(el)` check anyway.